sum-of-positive.py

```diff
@@ -0,0 +1,10 @@
+def positive_sum(arr):
+    """Add all positive numbers in list."""
+    if len(arr) == 0:
+        return 0
+    sum = 0
```

Comment:
Note that `sum` is already the name of a builtin function in Python... a very useful one for this solution in fact!

try this: `sum(n for n in arr if n > 0)`